components/brave_wallet_ui/common/hooks/transaction-parser.test.tsx

```diff
@@ -228,20 +251,42 @@ describe('useTransactionParser hook', () => {
          *   - gasLimit: 21000
          */
 
-        const { result: { current: transactionParser } } = renderHook(() => useTransactionParser(
-          mockNetwork,
-          [{
-            ...mockAccount,
-            tokenBalanceRegistry: {
-              [mockERC20Token.contractAddress.toLowerCase()]: '0'
-            },
-            address: '0xdeadbeef',
-            nativeBalanceRegistry: {
-              '0x1': '1000000000000000' // 0.001 ETH
-            }
-          }],
-          mockAssetPrices, [], []
-        ))
+        const customStore = createStore(combineReducers({
+          wallet: createWalletReducer({
+            ...mockWalletState,
+            transactionSpotPrices: mockAssetPrices,
+            fullTokenList: [...mockWalletState.fullTokenList, { ...mockERC20Token, contractAddress: '0xdeadbeef' }, mockERC20Token],
+            accounts: [
+              ...mockWalletState.accounts,
+              {
+                ...mockAccount,
+                tokenBalanceRegistry: {
+                  [mockERC20Token.contractAddress.toLowerCase()]: '0'
+                },
+                address: '0xdeadbeef',
+                nativeBalanceRegistry: {
+                  '0x1': '1000000000000000' // 0.001 ETH
+                }
+              }
+            ]
+          }),
+          page: createPageReducer(mockPageState)
+        }))
+
+        const { result: { current: transactionParser } } = renderHook(() => useTransactionParser(mockNetwork), renderHookOptionsWithCustomStore(customStore))
+        //   mockNetwork,
+        //   [{
+        //     ...mockAccount,
+        //     tokenBalanceRegistry: {
+        //       [mockERC20Token.contractAddress.toLowerCase()]: '0'
+        //     },
+        //     address: '0xdeadbeef',
+        //     nativeBalanceRegistry: {
+        //       '0x1': '1000000000000000' // 0.001 ETH
+        //     }
+        //   }],
+        //   mockAssetPrices, [], []
+        // ))
```

Comment:
Let's remove this if we don't need it anymore? Same with other occurrences in this file.

Reply:
removed